src/main/kotlin/com/terraformation/backend/accelerator/api/ModulesController.kt

```diff
@@ -0,0 +1,3 @@
+package com.terraformation.backend.accelerator.api
+
+class ModulesController {}
```

Comment:
Is it useful to merge this yet? Could wait to create it until it has some contents.